spec/system/relationship_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+RSpec.describe 'フォロー機能', type: :system do
+  let(:user) { FactoryBot.create(:user, profile: 'そろそろ眠いです') }
+  let(:michael) { FactoryBot.create(:user, name: 'michael', profile: '早起きが得意です', created_at: Time.zone.now) }
+  let(:tom) { FactoryBot.create(:user, name: 'tom', profile: 'お腹が空きました', created_at: 1.day.from_now) }
```

Comment:
ago / since で統一するとよいかも
```suggestion
  let(:tom) { FactoryBot.create(:user, name: 'tom', profile: 'お腹が空きました', created_at: 1.day.since) }
```